vrk-kpa/suomidigi
# SUOPA-716 Articles

To test: 
- `make fresh`
- Log in and go to https://suomidigi.docker.sh/admin/structure/taxonomy/manage/theme/overview and edit your favourite Theme tag. Add Theme description for the term and save the term.
- Create a new Landing page. https://suomidigi.docker.sh/node/add/landing_page
- Add a Views paragraph. Select "Articles" for view and "Articles | Grid" for display
- Save the node and clear all caches from the UI.
- Refresh the page and try to select different themes from the filter. Also check that your favourite theme tag shows the theme description.
